todo/material/task.md

```diff
@@ -0,0 +1,110 @@
+# Aufgabenliste
+
+::: info
+Diese Aufgabe basiert auf _keinem CS50 Practice Problem_.
+
+**Disclaimer:** Diese Aufgabe wurde nicht vom Lehrstuhl herausgegeben und kann Fehler enthalten. Sie dient, wie das gesamte Material von [inf-lab.dev](https://inf-lab.dev), lediglich zu Übungszwecken!
+:::
+
+Wir alle haben verschiedene Aufgaben, jedoch ist es nicht immer einfach sich alle seine Aufgaben zu merken. Daher wollen wir in dieser Aufgabe eine ToDo-Liste entwickeln, welche uns erlaubt unsere Aufgaben über die Konsole zu verwalten.
+
+## Teilaufgabe 1
+
+Wie immer, wenn wir einen vorbereiteten Code bekommen, sollten wir diesen vorerst verstehen bevor wir Änderungen vornehmen können. Lies deshalb den Code in der Datei `todo.py` und beantworte insbesondere folgende Fragen. Wenn du noch etwas unsicher mit der Python Syntax bist, kannst du die vorgegebene Datei `todo.c` zur hilfe nehmen. Diese enthält den selben Code - nur für die Programmiersprache C.
```

Comment:
I think it would improve the workflow in the labs to have the `wget` here as it not necessary to switch back to another view. Therefore, I would suggest to add this command here as well.

Reply:
While I agree, we should do this in some sort of smart way, because this introduces a big margin of error if we now have **two seperate** places where we need to pluck in the command to download.

So I see two options:
- just link to the `wget` command
- embed the partial "Aufgabenmaterial" page within a spoiler (this could be done using VitePress and some Vue+ CSS magic)
- embed the `wget` command only on the task and drop "Aufgabenmaterial" entirely

I want to avoid having to duplicate the command at all costs.

Reply:
I think we just need a code section which states the `wget` command. This should be placed at a suitable place in the task description.